app.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import path from 'path';
+import authRouter from './routes/auth';
+import storeRouter from './routes/store';
+import secretRouter from './routes/secret';
+import './models/index';
+
+const port = process.env.PORT || 3001;
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../client/build')))
+// app.use(express.static(path.join(__dirname, './build')))
```

Comment:
rules for commented code in frontend and backend are the same, so either remove or explain why is this here